File: scheduling/scheduling-annotation-processor/src/test/java/io/koraframework/scheduling/annotation/processor/controller/ScheduledWithTrigger.java
package io.koraframework.scheduling.annotation.processor.controller;

import org.quartz.JobExecutionContext;
import io.koraframework.scheduling.quartz.ScheduleWithTrigger;

public class ScheduledWithTrigger {
    @ScheduleWithTrigger(ScheduledWithTrigger.class)
    public void noArgs() {}

    @ScheduleWithTrigger(ScheduledWithTrigger.class)
    public void withCtx(JobExecutionContext jobExecutionContext) {}
}

File: scheduling/scheduling-ksp/src/test/kotlin/io/koraframework/scheduling/ksp/controller/ScheduledWithTrigger.kt
package io.koraframework.scheduling.ksp.controller

import org.quartz.JobExecutionContext
import io.koraframework.scheduling.quartz.ScheduleWithTrigger

class ScheduledWithTrigger {
    @ScheduleWithTrigger(ScheduledWithTrigger::class)
    fun noArgs() {
    }

    @ScheduleWithTrigger(ScheduledWithTrigger::class)
    fun withCtx(jobExecutionContext: JobExecutionContext?) {
    }
}
